docs/Accessibility.md

```diff
@@ -0,0 +1,75 @@
+
+#### accessibilityTraits
+
+Accessibility traits tell a person using VoiceOver what kind of element they have selected. Is this element a label? A button? A header? These questions are answered by `accessibilityTraits`.
+
+To use, set the `accessibilityTraits` property to one of (or an array of) accessibility trait strings:
+
+* **none** Used when the element has no traits.
+* **button** Used when the element should be treated as a button.
+* **link** Used when the element should be treated as a link.
+* **header** Used when an element acts as a header for a content section (e.g. the title of a navigation bar).
+* **search** Used when the text field element should also be treated as a search field.
+* **image** Used when the element should be treated as an image. Can be combined with button or link, for example.
+* **selected**  Used when the element is selected. For example, a selected row in a table or a selected button within a segmented control.
+* **plays** Used when the element plays its own sound when activated.
+* **key** Used when the element acts as a keyboard key.
+* **text** Used when the element should be treated as static text that cannot change.
+* **summary** Used when an element can be used to provide a quick summary of current conditions in the app when the app first launches.  For example, when Weather first launches, the element with today's weather conditions is marked with this trait.
+* **disabled** Used when the control is not enabled and does not respond to user input.
+* **frequentUpdates** Used when the element frequently updates its label or value, but too often to send notifications. Allows an accessibility client to poll for changes. A stopwatch would be an example.
+* **startsMedia** Used when activating an element starts a media session (e.g. playing a movie, recording audio) that should not be interrupted by output from an assistive technology, like VoiceOver.
+* **adjustable** Used when an element can be "adjusted" (e.g. a slider).
+* **allowsDirectInteraction** Used when an element allows direct touch interaction for VoiceOver users (for example, a view representing a piano keyboard).
+* **pageTurn** Informs VoiceOver that it should scroll to the next page when it finishes reading the contents of the element.
+
+#### onAccessibilityTap
+
+Use this property to assign a custom function to be called when someone activates an accessible element by double tapping on it while it's selected.
+
+#### onMagicTap
+
+Assign this property to a custom function which will be called when someone performs the "magic tap" gesture, which is a double-tap with two fingers. A magic tap function should perform the most relevant action a user could take on a component. In the Phone app on iPhone, a magic tap answers a phone call, or ends the current one.
```

Comment:
When you perform the magic tap gesture, it starts at the selected element and bubbles up through the parents to see if anyone implements the magic tap gesture.